ansible/private-docker-registry/roles/docker-registry/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: create directory for self-signed SSL cert
+  file: path=/var/registry/certs state=directory
+
+- name: create self-signed cfssl json file
+  template:
+     src: "{{ role_path }}/templates/self-csr.json.j2"
+     dest: ./self-csr.json
+
+- name: get cfssl
+  get_url:
+    url: https://pkg.cfssl.org/R1.2/cfssl_linux-amd64
+    dest: ./cfssl
+    mode: 0755
+
+- name: get cfssljson
+  get_url:
+    url: https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64
+    dest: ./cfssljson
+    mode: 0755
+
+- name: gencert
+  shell: ./cfssl gencert -initca self-csr.json | ./cfssljson -bare ca
```

Comment:
Yes.  It doesn't hurt anything.  It just creates a new cert.  This playbook is not designed to maintain a private docker registry.  It is just for standing one up.